lib/src/disposable/disposable.dart

```diff
@@ -427,6 +428,28 @@ class Disposable implements _Disposable, DisposableManagerV3 {
     return null;
   }
 
+  void subscribeToDocumentEvent(String event, EventListener callback,
```

Comment:
👼 I would make a note to include doc comments when the `DisposableManager` interface is updated.